<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppQtMainWidget</class>
 <widget class="QMainWindow" name="AppQtMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>AusweisApp2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/ausweisapp.qrc">
    <normaloff>:/images/npa.ico</normaloff>:/images/npa.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <widget class="QWidget" name="appMainWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="headerWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>10</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="centralWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="appWidget">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="governikus::TabButtonGroup" name="mainTabList" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="accessibleName">
             <string>Actions</string>
            </property>
            <layout class="QVBoxLayout" name="menuLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="startPushButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Welcome</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="governikus::TabButton" name="ausweisenToolButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">margin-top: 21px;</string>
               </property>
               <property name="text">
                <string> Identify</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="governikus::TabButton" name="providerToolButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string> Provider</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoRepeatDelay">
                <number>300</number>
               </property>
               <property name="autoRepeatInterval">
                <number>100</number>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="governikus::TabButton" name="historyToolButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string> History</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="governikus::TabButton" name="settingsToolButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string> Settings</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>15</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="betaLabel">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="applicationPage">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QWidget" name="widget" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <layout class="QGridLayout" name="applicationGridLayout">
                   <item row="3" column="0">
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item row="3" column="2">
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item row="3" column="1">
                    <widget class="QLabel" name="appLabel">
                     <property name="focusPolicy">
                      <enum>Qt::StrongFocus</enum>
                     </property>
                     <property name="accessibleName">
                      <string>nPA and eAT Logo</string>
                     </property>
                     <property name="pixmap">
                      <pixmap resource="../../resources/ausweisapp.qrc">:/images/start_nPA_eAT.png</pixmap>
                     </property>
                    </widget>
                   </item>
                   <item row="4" column="1">
                    <spacer name="verticalSpacer">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item row="0" column="1">
                    <spacer name="verticalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLabel" name="appLogoLabel">
                     <property name="focusPolicy">
                      <enum>Qt::StrongFocus</enum>
                     </property>
                     <property name="accessibleName">
                      <string>AusweisApp2 Logo</string>
                     </property>
                     <property name="pixmap">
                      <pixmap resource="../../resources/ausweisapp.qrc">:/images/AppLogo_AutentApp2_2014.png</pixmap>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <spacer name="verticalSpacer_5">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="governikus::ProviderWidget" name="providerPage"/>
            <widget class="governikus::SelfInformationWidget" name="ausweisenPage">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="governikus::HistoryWidget" name="historyPage"/>
            <widget class="governikus::SettingsWidget" name="settingsPage"/>
            <widget class="QWidget" name="workflowPage">
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>6</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="appLogoWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="logoLabel">
          <property name="pixmap">
           <pixmap resource="../../resources/ausweisapp.qrc">:/images/Logo_AutentApp2_2014.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="governikus::DeveloperModeHistoryWidget" name="developerModeHistoryWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>170</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionAusweisen"/>
    <addaction name="actionProvider"/>
    <addaction name="actionHistory"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionSetupAssistant"/>
    <addaction name="actionDiagnosis"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionQuestion"/>
    <addaction name="actionSendError"/>
    <addaction name="actionEvaluate"/>
    <addaction name="separator"/>
    <addaction name="actionSaveProtocol"/>
    <addaction name="actionShowProtocol"/>
    <addaction name="separator"/>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuPINManagement">
    <property name="title">
     <string>&amp;PIN Management</string>
    </property>
    <addaction name="actionChangePin"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuPINManagement"/>
   <addaction name="menuHilfe"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Minimise</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAusweisen">
   <property name="text">
    <string>&amp;Identify</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Manual</string>
   </property>
  </action>
  <action name="actionEvaluate">
   <property name="text">
    <string>&amp;Evaluate</string>
   </property>
  </action>
  <action name="actionSendError">
   <property name="text">
    <string>&amp;Report error</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>&amp;About AusweisApp2</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionProvider">
   <property name="text">
    <string>&amp;Provider</string>
   </property>
  </action>
  <action name="actionShowProtocol">
   <property name="text">
    <string>S&amp;how log</string>
   </property>
  </action>
  <action name="actionSaveProtocol">
   <property name="text">
    <string>Save &amp;log</string>
   </property>
  </action>
  <action name="actionQuestion">
   <property name="text">
    <string>&amp;Questions</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>&amp;History</string>
   </property>
  </action>
  <action name="actionChangePin">
   <property name="text">
    <string>&amp;Change PIN</string>
   </property>
  </action>
  <action name="actionSetupAssistant">
   <property name="text">
    <string>&amp;Setup assistant</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDiagnosis">
   <property name="text">
    <string>&amp;Diagnosis</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>governikus::TabButtonGroup</class>
   <extends>QWidget</extends>
   <header>generic/TabButtonGroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>governikus::SettingsWidget</class>
   <extends>QWidget</extends>
   <header>SettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>governikus::ProviderWidget</class>
   <extends>QWidget</extends>
   <header>ProviderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>governikus::SelfInformationWidget</class>
   <extends>QWidget</extends>
   <header>SelfInformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>governikus::HistoryWidget</class>
   <extends>QWidget</extends>
   <header>HistoryWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>governikus::DeveloperModeHistoryWidget</class>
   <extends>QWidget</extends>
   <header>DeveloperModeHistoryWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>governikus::TabButton</class>
   <extends>QToolButton</extends>
   <header>generic/TabButtonGroup.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>startPushButton</tabstop>
  <tabstop>appLogoLabel</tabstop>
  <tabstop>appLabel</tabstop>
  <tabstop>ausweisenToolButton</tabstop>
  <tabstop>providerToolButton</tabstop>
  <tabstop>historyToolButton</tabstop>
  <tabstop>settingsToolButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/ausweisapp.qrc"/>
 </resources>
 <connections/>
</ui>
